lib/apachetest.pm

```diff
@@ -346,7 +346,7 @@ sub postgresql_cleanup {
     systemctl 'stop postgresql';
     systemctl 'disable postgresql';
     systemctl 'is-active postgresql', expect_false => 1;
-    assert_script_run q{kill -s KILL $(ps -u postgres -o pid=)};
+    assert_script_run('kill -s KILL $(ps -u postgres -o pid=)') unless script_run('ps -u postgres -o pid=');
```

Comment:
line 346 should deal with postgresql server, as it should stop upon received signal, and postmaster process should be dead by then.